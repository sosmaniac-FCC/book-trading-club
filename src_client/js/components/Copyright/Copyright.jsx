import React from 'react';

const Copyright = () => (
    <div>
        <div className="container">
            <div className="row">
                <div className="left">
                    <h5 className="white-text flow-text">Book Trading Club</h5>
                    <p className="grey-text text-lighten-3">Collect & trade books</p>
                </div>
                <div className="right">
                    <h5 className="white-text flow-text">FreeCodeCamp <i className="fa fa-free-code-camp fa-1x"></i></h5>
                    <p className="green-text text-darken-3 right" href="https://www.freecodecamp.org/sosmaniac-fcc" target="_blank">@sosmaniac-FCC</p>
                </div>
            </div>
        </div>
        <div className="footer-copyright purple accent-3">
            <div className="container">
                <p className="black-text left">Styled with Materialize</p>
                <p className="black-text right">Coded by John Simmons</p>
            </div>
        </div>
    </div>
);

export default Copyright;