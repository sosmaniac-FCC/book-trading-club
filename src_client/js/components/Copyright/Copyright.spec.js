// expect from jest_expect.js
/* global expect */

import React from 'react';
import { shallow, configure } from 'enzyme';
import enzymeAdapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';

configure({ adapter: new enzymeAdapter() });

import Copyright from './Copyright.jsx';

test('Copyright section below the router content loads as expected.', () => {
    const component = shallow(<Copyright />);
    const tree = toJSON(component);
    expect(tree).toMatchSnapshot();
});

test('Copyright section should have no style attributes.', () => {
    const component = shallow(<Copyright />);
    expect(component.find('[style]')).toHaveLength(0);
});