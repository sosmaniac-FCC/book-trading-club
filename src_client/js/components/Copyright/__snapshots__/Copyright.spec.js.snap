// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright section below the router content loads as expected. 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="left"
      >
        <h5
          className="white-text flow-text"
        >
          Book Trading Club
        </h5>
        <a
          className="grey-text text-lighten-3"
        >
          Collect & trade books
        </a>
      </div>
      <div
        className="right"
      >
        <h5
          className="white-text flow-text"
        >
          FreeCodeCamp 
          <i
            className="fa fa-free-code-camp fa-1x"
          />
        </h5>
        <a
          className="green-text text-darken-3 right"
          href="https://www.freecodecamp.org/sosmaniac-fcc"
          target="_blank"
        >
          @sosmaniac-FCC
        </a>
      </div>
    </div>
  </div>
  <div
    className="footer-copyright purple accent-3"
  >
    <div
      className="container"
    >
      <a
        className="black-text left"
      >
        Styled with Materialize
      </a>
      <a
        className="black-text right"
      >
        Coded by John Simmons
      </a>
    </div>
  </div>
</div>
`;
